CONTRIBUTING.md

```diff
@@ -34,6 +39,7 @@ Contributions are welcome! Here's how you can help:
 5. Once you are happy with your changes, submit a pull request.
      - Open the [pull-request form](https://github.com/minetest/minetest/pull/new/master).
      - Add a description explaining what you've done (or if it's a work-in-progress - what you need to do).
+     - Don't put many commits into a single pull request. Large pull requests are hard or even impossible to review and the individual commits don't get tested by the continuous integration, just the last one is.
```

Comment:
I'd prefer a positive sentence here:
`- The commits should be easy to overview. Consider squashing similar commits or splitting the PR to a new one.`

Why would you want to test the individual commits? You usually add new commits to fix issues/errors or do further functional improvements, and CI reruns when you force-push. Thus, the second sentence does not make sense for me.

Reply:
...And I'm just sitting here, looking for the missing space after the end of each sentence.
`...and splitting the PR into a new one.`
`overview` is awkward, but it's your technical term and is likely the most appropriate.

Reply:
It's not really the number of commits in a PR that is a problem but the overall size of the PR.
I don't like the wording here.

Reply:
> Why would you want to test the individual commits?

An "inner commit" in a multi-commit PR could break the build and the CI won't catch that. Only the last commit in a PR is required to do everything right. When such a multi-commit PR gets pulled in and later someone tries to hunt down a bug using git bisect, he may encounter the broken commit, have to use git bisect skip and possibly end up dealing with the broken commit + another commit on top of it combo when the bug happens to be somewhere around the broken commit.

Reply:
I removed that sentence about CI from the new version though.